connectors/sources/jira.py

```diff
@@ -56,11 +61,43 @@
     ISSUE_DATA: "/rest/api/2/issue/{id}",
     ATTACHMENT_CLOUD: "/rest/api/2/attachment/content/{attachment_id}",
     ATTACHMENT_SERVER: "/secure/attachment/{attachment_id}/{attachment_name}",
+    USERS: "/rest/api/3/users/search",
+    PROJECT_PERMISSIONS: "/rest/api/2/user/permission/search?projectKey={project_key}&permissions=BROWSE_PROJECTS",
+    PROJECT_ROLE_MEMBERS: "/rest/api/3/project/{project_key}/role/{role_id}",
+    ISSUE_SECURITY_LEVEL: "/rest/api/2/issue/{issue_key}?fields=security",
+    SECURITY_LEVEL_MEMBERS: "/rest/api/3/issuesecurityschemes/level/member?maxResults={max_results}&startAt={start_at}&levelId={level_id}&expand=user,group,projectRole",
```

Comment:
suggestion: I think functions called `project_permissions`, `project_role_members` etc. would be more explicit. Here it's not obvious that these format string constants expect arguments. A function/method would make it obivous that something needs to be passed (and especially what).

Reply:
Sure, I'll tweak a name of these constants. Renaming them to `PROJECT_PERMISSIONS_BY_KEY` and `PROJECT_ROLE_MEMBERS_BY_ROLE_ID`.

Reply:
I would still prefer methods as nearly every mature IDE can easily autocomplete method parameters. I think a formatted string doesn't have a clear interface, which makes it harder to understand. Having FF in mind I can definitely live with the current solution, but would prefer creating a follow-up issue for making it cleaner using explicit methods.

Reply:
Sure, even feel free to add it in the existing issue https://github.com/elastic/connectors-python/issues/1417 👍